packages/use-presence/src/use-presence-switch.tsx

```diff
@@ -0,0 +1,35 @@
+import {useState, useEffect} from 'react';
+import usePresence from './use-presence';
+
+export function usePresenceSwitch<ItemType>(
+  latestItem: ItemType,
```

Comment:
Maybe just `item`? I'd care more about the naming of this over the internal state variable, as this is what the consumer will see in autocomplete.

Reply:
`item` & `mountedItem` works fine for me 👍